bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -44,8 +44,6 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
     HttpSession session = request.getSession(false);
```

Comment:
I can't comment higher up, but this class should have a java doc.

Reply:
Added javadoc